File: hw/pastor/u1/hw06Dependencies/chickenInJavaScript/ChickenFarmSimulator.view.js
const  Chicken  = require('./Chicken.js');

const readline = require('readline');

console.log("This is my chicken Farm Simulator by Mathews Pastor");

const owner = "Mathews Pastor";
let id = 1;
const name = "Lucy";
const color = "White and Brown";
let age = 2;
let isMolting = false;

let chicken = new Chicken(id, name, color, age, isMolting);

console.log(`The chicken is: ${chicken.toString()}`);

File: hw/quina/u1/HW06-Dependency/ChickenFarmSimulatorPython/ChickenFarmSimulator.view.py
from Chicken import Chicken

if __name__ == "__main__":
    print("\nThis is my Chicken Farm Simulator by Maryuri")
    print("The Chicken is\n")
    
    chicken = Chicken(1, "Lucy", "White and Brown", False, 2)
    print(chicken)
    
    chicken.set_color("Brown")